dockstore-integration-testing/src/test/java/io/dockstore/client/cli/EntryResourceIT.java

```diff
@@ -139,23 +142,90 @@ public void testOrcidExport() {
             hoverfly.resetState();
             // Manually change it to the wrong put code
             testingPostgres.runUpdateStatement(
-                    String.format("update workflow set orcidputcode='%s'where orcidputcode='%s'", BAD_PUT_CODE, PUT_CODE));
+                    String.format("update entry_orcidputcode set orcidputcode='%s' where orcidputcode='%s'", BAD_PUT_CODE, PUT_CODE_USER_1));
             // Dockstore should be able to recover from having the wrong put code for whatever reason
             entriesApi.exportToORCID(workflowId, null);
 
-            // Clear DB
-            testingPostgres.runUpdateStatement("update workflow set orcidputcode=null");
+            // Remove the put code. Test scenario where the put code and DOI URL are on ORCID, but the put code is not on Dockstore
```

Comment:
~Just to confirm my understanding, this PR doesn't seem to change existing hoverfly code.~
But hoverfly is providing an "existing" putcode which is why this passes?

Nvm, saw the hoverfly code below, but still a little curious